docs/overview/introduction.md

```diff
@@ -24,29 +24,37 @@ sidebar_position: 1
 1. Kubernetes
 1. Deployment + NGINX
 
-## Importance of containerization and orchestration
+## What is this course about?
 
-### Containerization in a nutshell
+The goal of this course is to teach you about **containers** and **orchestration with Kubernetes**.
+After this course, you will be able to deploy an application into production that can serve users
+worldwide with minimal latency and headache for you, and the development team (which may include you
+😎).
 
-### Why would someone care about containerization?
+ In brief, containerization is packaging an application and its dependencies into a standardized
+ unit, known as a container. This container can then be easily moved between environments, ensuring
+ consistency and reliability across different systems.
 
-Because we care about reproducible environments. Let's say you are developing a microservice
-application. So each component of your system resides in a separate, isolated environment. This
-means that if, for whatever reason, a runtime error occurs in your application, then that error
-is contained and the rest of the system remains unaffected.
+ We care about containerization because it allows developers to build, test, and deploy applications
+ more efficiently. By isolating applications within containers, we can avoid conflicts between
+ dependencies and ensure that the application runs consistently regardless of the underlying
+ infrastructure.
 
-Another important reason is we can use containers to package and deliver applications with an
-environment that contains all the required dependencies for them to function properly. This
-eliminates the problem of running an application on multiple systems, each of them with different
-versions of dependencies. The classic *works on my system* problem.
+ Orchestration is managing and coordinating multiple containers in a distributed environment. This
+ includes tasks such as scaling containers up or down, load balancing traffic between containers,
+ and ensuring high availability of the application. Kubernetes is a popular orchestration tool that
+ helps automate these tasks, making it easier to manage containerized applications at scale.
 
-### What does it mean to orchestrate some containers?
+ By the end of this course, you will have the skills and knowledge needed to deploy and manage
+ containerized applications in a production environment.
```

Comment:
Remove extra whitespace at the beginning of each line

Reply:
Maybe also justify the content if it looks right (i.e. the space between words does not get extremely stretched)

Reply:
I'm not actually sure how to do that? I thought it was due to the word-wrapping, but even after putting it all in one line it still puts a lot of space between words